python/lsst/ap/association/loadDiaCatalogs.py

```diff
@@ -192,6 +197,35 @@ def loadDiaSources(self, diaObjects, pixelRanges, dateTime, apdb):
                              inplace=True)
         return diaSources.replace(to_replace=[None], value=np.nan)
 
+    @pipeBase.timeMethod
+    def loadDiaForcedSources(self, diaObjects, dateTime, apdb):
+        """Load DiaObjects from the Apdb based on their HTM location.
+
+        Parameters
+        ----------
+        pixelRanges : `tuple` [`int`]
+            Ranges of pixel values that cover region of interest.
```

Comment:
This looks out of sync with the function params

Reply:
Sorry! Copy paste bites me again.